actions/workflows/create_vm.yaml

```diff
@@ -2,17 +2,15 @@
   chain:
     -
       name: "get_subnet_id"
-      ref: "st2cd.kvstore"
+      ref: "st2.kv.get"
```

Comment:
In that case please make sure that works to and all this works e2e on st2build002. Often its simpler to handle that as a separate task but that is totally upto @emedvedev